pages/installation/os/sles.rst

```diff
@@ -0,0 +1,114 @@
+*******************
+SLES installation
+*******************
+
+This guide describes the fastest way to install Graylog on SLES 12 SP3. All links and packages are present at the time of writing but might need to be updated later on.
+
+.. warning:: This setup should not be done on publicly exposed servers. This guide **does not cover** security settings!
+
+
+Prerequisites
+-------------
+
+he following patterns are required for a minimal setup (see `SLES 12 SP3 Deployment Guide <https://www.suse.com/documentation/sles-12/singlehtml/book_sle_deployment/book_sle_deployment.html#sec.i.yast2.proposal.sofware>`_)::
+
+  - Base System
+  - Minimal System (Appliances)
+  - YaST configuration packages
+
+.. warning:: This Guide assumes that the firewall is disabled and communication is possible to the outside world.
+
+Assuming a minimal setup, you will need to install these additional packages::
```

Comment:
Minor nitpick: "These packages" when it's only a single package.
We could simply say:
> Assuming a minimal setup, you have to install the Java runtime environment::